docs/administrator-handbook.md

```diff
@@ -68,13 +78,30 @@ Will download the target package, its dependencies, and create a new deployment
 with those packages installed.  It is also possible to specify a local package
 which is not part of a repository.
 
-To remove layered packages installed from a repository, use `rpm-ostree uninstall
-<pkg>`.  To remove layered packages installed from a local package, you must
-specify the full NEVRA of the package.  For example, `rpm-ostree uninstall
-ltrace-0.7.91-16.fc22.x86_64`.
+To remove layered packages installed from a repository, use:
 
-In order to uninstall a package that is a part of the base layer, use `rpm-ostree override remove <pkg>`.
-For example, `rpm-ostree override remove firefox`.
+```
+# rpm-ostree uninstall <pkg>
+```
+
+To remove layered packages installed from a local package, you must
+specify the full NEVRA of the package. For example:
+
+```
+# rpm-ostree uninstall ltrace-0.7.91-16.fc22.x86_64
+```
```

Comment:
Ahh heh, this is actually no longer true as of #1400. Maybe just drop that hunk? Or feel free to leave as is for now since it's not new and we can address it as a follow-up!